kin-devplatform-sdk/src/main/java/kin/devplatform/history/view/OrderHistoryRecyclerAdapter.java

```diff
@@ -125,8 +125,8 @@ private boolean amITheRecipientOfOrder(Order item) {
 				case SPEND:
 					return false;
 				case PAY_TO_USER:
-					String publicAddress = BlockchainSourceImpl.getInstance().getPublicAddress();
-					return item.getBlockchainData().getRecipientAddress().equals(publicAddress);
+						String publicAddress = BlockchainSourceImpl.getInstance().getPublicAddress();
+						return item.getBlockchainData().getRecipientAddress().equals(publicAddress);
```

Comment:
Same comment regarding the code formatting

Reply:
:+1: